ProcessLib/HeatTransportBHE/CreateHeatTransportBHEProcess.cpp

```diff
@@ -139,8 +146,82 @@ std::unique_ptr<Process> createHeatTransportBHEProcess(
     auto media_map =
         MaterialPropertyLib::createMaterialSpatialDistributionMap(media, mesh);
 
-    HeatTransportBHEProcessData process_data{std::move(media_map),
-                                             std::move(bhes)};
+    // find if bhe uses python boundary condition
+    auto const isUsingPythonBC =
+        visit([](auto const& bhe) { return bhe.usePythonBC; }, bhes[0]);
+    if (isUsingPythonBC)
+    {
+        has_network_python_bc = true;
+    }
+
+    //! Python object computing BC values.
+    BHEInflowPythonBoundaryConditionPythonSideInterface* py_object = nullptr;
+    // create a pythonBoundaryCondition object
+    if (has_network_python_bc)
```

Comment:
same as
```c++
if (is_using_python_bc)
```
then the `has_network_python_bc` can be removed.

Reply:
Fixed in PR#2793.